### DISCUSSION TITLE
How to change default language?

### DISCUSSION THREAD

**Discussion by gniyliu** [mediacms-io/mediacms#727] [Q&A]
I want to change default language(EN) to another one, how to do that?

**Comment by mgogoulos** [comment]
It's just English, MediaCMS is not translated to other languages

**Reply by gniyliu** [reply]
Thanks for you answer. 
I asked chatGPT, it said MediaCMS can support multi-language, but must compile the frontend code.
Maybe chapGPT just made a mistake?

**Reply by mgogoulos** [reply]
Hi, there have been efforts in the past to translate it, but unfortunately, they didn't find their way to the software...yet. It needs a final push to get there, I know it's something important

**Reply by gniyliu** [reply]
Ok, I think MediaCMS is a great project, and I'll follow this.
